build.gradle.kts

```diff
@@ -0,0 +1,79 @@
+plugins {
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "0.1.0"
+}
+
+configurations {
+  implementation { exclude(mapOf("module" to "tomcat-jdbc")) }
+}
+
+extra["spring-security.version"] = "5.3.1.RELEASE" // Updated since spring-boot-starter-oauth2-resource-server-2.2.5.RELEASE only pulls in 5.2.2.RELEASE (still affected by CVE-2018-1258 though)
+
+dependencies {
+  annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
+  annotationProcessor("org.projectlombok:lombok:1.18.12")
+
+  testAnnotationProcessor("org.projectlombok:lombok:1.18.12")
+
+  implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+  implementation("org.springframework.boot:spring-boot-starter-cache")
+  implementation("org.springframework.boot:spring-boot-starter-quartz")
+  implementation("org.springframework.boot:spring-boot-starter-security")
+  implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
+  implementation("org.springframework.boot:spring-boot-starter-oauth2-client")
+
+  implementation("org.springframework.security:spring-security-oauth2-jose")
+  implementation("org.springframework:spring-webflux")
+  implementation("org.springframework.boot:spring-boot-starter-reactor-netty")
+
+  implementation("org.apache.camel:camel-spring-boot:2.25.1")
+  implementation("org.apache.camel:camel-quartz2:2.25.1")
+
+  implementation("org.springframework:spring-jms")
+  implementation("com.amazonaws:amazon-sqs-java-messaging-lib:1.0.8")
+
+  implementation("org.hibernate:hibernate-core")
```

Comment:
spotted that we were using `java8` here by accident - been deprecated by `core` now for a few years - https://stackoverflow.com/questions/39761619/why-is-hibernate-java8-hibernate-5-x-jar-shown-deprecated